src/rgb-color.js

```diff
@@ -36,49 +38,53 @@ class RGBColor {
       const def = colorDefs[i];
       const bits = def.re.exec(colorString);
       if (bits) {
-        ([this.r, this.g, this.b] = def.process(bits));
+        [this.r, this.g, this.b, this.a] = def.process(bits);
         this.ok = true;
       }
     }
 
     // validate/cleanup values
-    if (this.r < 0 || Number.isNaN(this.r) || this.r === undefined) {
+    if (this.r < 0 || Number.isNaN(this.r) || this.r == null) {
       this.r = 0;
     } else if (this.r > 255) {
       this.r = 255;
     }
-    if (this.g < 0 || Number.isNaN(this.g) || this.g === undefined) {
+    if (this.g < 0 || Number.isNaN(this.g) || this.g == null) {
       this.g = 0;
     } else if (this.g > 255) {
       this.g = 255;
     }
-    if (this.b < 0 || Number.isNaN(this.b) || this.b === undefined) {
+    if (this.b < 0 || Number.isNaN(this.b) || this.b == null) {
       this.b = 0;
     } else if (this.b > 255) {
       this.b = 255;
     }
+    if (this.a < 0 || Number.isNaN(this.a)) {
```

Comment:
I am not sure if this step is necessary here or not. Anyway, this is where coverage is lacking and I wasn't sure how I should test it.